examples/aggregated_k8s/agg.yaml

```diff
@@ -23,143 +22,94 @@ metadata:
   name: vllm-agg
 spec:
   services:
-    ModelExpressServer:
+    ModelExpress:
+      envFromSecret: hf-token-secret
       livenessProbe:
         tcpSocket:
-          port: 8002
+          port: 8000
         initialDelaySeconds: 30
         periodSeconds: 10
         timeoutSeconds: 5
         failureThreshold: 3
       readinessProbe:
         tcpSocket:
-          port: 8002
+          port: 8000
         initialDelaySeconds: 10
         periodSeconds: 5
         timeoutSeconds: 3
         failureThreshold: 3
       dynamoNamespace: vllm-agg
-      componentType: main
+      componentType: frontend
       replicas: 1
       resources:
         requests:
-          cpu: "500m"
-          memory: "256Mi"
+          cpu: "4"
+          memory: "16Gi"
         limits:
-          cpu: "500m"
-          memory: "256Mi"
+          cpu: "4"
+          memory: "16Gi"
       extraPodSpec:
         mainContainer:
-          image: model-express:latest
+          image: nvcr.io/nvidia/ai-dynamo/modelexpress-server:0.2.0
           imagePullPolicy: IfNotPresent
           env:
             - name: MODEL_EXPRESS_SERVER_PORT
-              value: "8002"
+              value: "8000"
             - name: MODEL_EXPRESS_LOGGING_LEVEL
               value: "info"
             - name: MODEL_EXPRESS_DATABASE_PATH
               value: "/root/models.db"
             - name: MODEL_EXPRESS_CACHE_DIRECTORY
-              value: "/root/.model-express/cache"
+              value: "/model/.model-express/cache"
             - name: MODEL_NAME
               value: "Qwen/Qwen3-0.6B"
             - name: MODEL_CACHE_PATH
-              value: "/root/.model-express/cache"
+              value: "/model/.model-express/cache"
+            - name: HF_HUB_CACHE
+              value: "/model/.model-express/cache"
           command:
             - /bin/sh
             - -c
           args:
             - |
               echo "Starting Model Express Server..."
               ./modelexpress-server &
+              
               SERVER_PID=$!
               echo "Server started with PID: $SERVER_PID"
 
               echo "Setting up Model Express configuration..."
               mkdir -p $MODEL_CACHE_PATH
+              mkdir -p /root/.model-express
```

Comment:
is this intended? the cache dir seems to be under `/model` but we're creating config under `/root`

Reply:
moved to /model as we caught in benchmarking - good catch